Windows node pool (#2130)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/azure/aks.md

6. Enter the **Managed Disk** information and its size.

   :::info

>    Additional worker node pools can be added after creating the system node pool if you want to customize specific
>    worker nodes for specialized workloads. For example, the system worker pool can be configured with the
>    _Standard_D2_v2_ instance type for general-purpose workloads, while another worker pool with the _Standard_NC12s_v3_
>    instance type can be configured for GPU workloads.
> 
>    You can also select **OS Type** as **Windows** to create a worker pool specifically for Windows workloads.

Reviewer: ```suggestion
   You can add more worker node pools after creating the system node pool to customize specific
   worker nodes for specialized workloads. For example, you can configure the system worker pool with the
   _Standard_D2_v2_ instance type for general-purpose workloads, and another worker pool with the _Standard_NC12s_v3_
   instance type for GPU workloads.

   You can also select **OS Type** as **Windows** to create a worker pool specifically for Windows workloads.
```

:::info

> New worker pools may be added if you want to customize specific worker nodes for specialized workloads. As an example,
> the default worker pool may be configured with the <i>Standard_D2_v2</i> instance type for general-purpose workloads,
> and another worker pool with the instance type <i>Standard_NC12s_v3</i> can be configured for GPU workloads.

Reviewer: ```suggestion
You can add new worker pools may to customize specific worker nodes for specialized workloads. As an example, you can configure
the default worker pool with the _Standard_D2_v2_ instance type for general-purpose workloads,
and another worker pool with the _Standard_NC12s_v3_ instance type for GPU workloads.
```
